### ISSUE TITLE
build: fix autotools warnings

### ISSUE THREAD

**Issue by garlick** [LLNL/lmt#33]
```
$ ./autogen.sh
configure.ac:60: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2031: AC_CACHE_VAL is expanded from...
../../lib/autoconf/general.m4:2052: AC_CACHE_CHECK is expanded from...
config/x_ac_check_cond_lib.m4:23: X_AC_CHECK_COND_LIB is expanded from...
configure.ac:60: the top level
[snip]
```

This is with `autoconf-2.69-11.el7.noarch` on TOSS 3.4

**Comment by ofaaland** [comment]
@garlick thanks

**Comment by ofaaland** [comment]
Fixed by #42
